nvme-wrap.c

```diff
@@ -95,6 +95,39 @@ int nvme_cli_identify_allocated_ns_list(struct nvme_dev *dev, __u32 nsid,
 	return do_admin_op(identify_allocated_ns_list, dev, nsid, list);
 }
 
+int nvme_cli_identify_secondary_ctrl_list(struct nvme_dev *dev, __u32 nsid, __u16 ctrl_id,
```

Comment:
Can you do this with a wrapper in the same style as the rest of this file? If the underlying call isn't available in libnvme, we should add the identify logic there instead.

Reply:
The libnvme misses nvme_identify(fd, ...), and libnvme-mi misses nvme_mi_mi_admin_identify_secondary_ctrl_list(). 

We need either of these to simplify the logic here. 

But sure, we can make changes in the lib instead of cli.

Reply:
I'm happy to add those to libnvme if you like!

Reply:
That would be great.

Reply:
OK, PR sent:

https://github.com/linux-nvme/libnvme/pull/480

I haven't tested this with nvme-cli, as that's what you're proposing here. Let me know how it goes!